Flaky test: Test_Subscriber_StartSyncsNewProposals

```
--- FAIL: Test_Subscriber_StartSyncsNewProposals (2.01s)
    repository_test.go:88:
        	Error Trace:	repository_test.go:88
        	Error:      	Condition never satisfied
        	Test:       	Test_Subscriber_StartSyncsNewProposals
    repository_test.go:89:
        	Error Trace:	repository_test.go:89
        	Error:      	Not equal:
        	            	expected: []market.ServiceProposal{market.ServiceProposal{ID:0, Format:"", ServiceType:"mock_service", ServiceDefinition:brokerdiscovery.mockServiceDefinition{}, PaymentMethodType:"mock_payment", PaymentMethod:brokerdiscovery.mockPaymentMethod{}, ProviderID:"0x1", ProviderContacts:market.ContactList{market.Contact{Type:"mock_contact", Definition:brokerdiscovery.mockContact{}}}, AccessPolicies:(*[]market.AccessPolicy)(nil)}}
        	            	actual  : []market.ServiceProposal{}

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,21 +1,2 @@
        	            	-([]market.ServiceProposal) (len=1) {
        	            	- (market.ServiceProposal) {
        	            	-  ID: (int) 0,
        	            	-  Format: (string) "",
        	            	-  ServiceType: (string) (len=12) "mock_service",
        	            	-  ServiceDefinition: (brokerdiscovery.mockServiceDefinition) {
        	            	-  },
        	            	-  PaymentMethodType: (string) (len=12) "mock_payment",
        	            	-  PaymentMethod: (brokerdiscovery.mockPaymentMethod) {
        	            	-  },
        	            	-  ProviderID: (string) (len=3) "0x1",
        	            	-  ProviderContacts: (market.ContactList) (len=1) {
        	            	-   (market.Contact) {
        	            	-    Type: (string) (len=12) "mock_contact",
        	            	-    Definition: (brokerdiscovery.mockContact) {
        	            	-    }
        	            	-   }
        	            	-  },
        	            	-  AccessPolicies: (*[]market.AccessPolicy)(<nil>)
        	            	- }
        	            	+([]market.ServiceProposal) {
        	            	 }
        	Test:       	Test_Subscriber_StartSyncsNewProposals
```

--- a/core/discovery/brokerdiscovery/repository_test.go
+++ b/core/discovery/brokerdiscovery/repository_test.go
@@ -76,7 +76,7 @@ func Test_Subscriber_StartSyncsNewProposals(t *testing.T) {
 	connection := nats.StartConnectionMock()
 	defer connection.Close()
 
-	repo := NewRepository(connection, NewStorage(eventbus.New()), 10*time.Millisecond, 10*time.Millisecond)
+	repo := NewRepository(connection, NewStorage(eventbus.New()), 10*time.Millisecond, 1*time.Second)
 	err := repo.Start()
 	defer repo.Stop()
 	assert.NoError(t, err)

